src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -158,6 +164,18 @@ Blank()
 >> DateTimeValue(ParseJSON("""abcdef"""))
 #Error(Kind=InvalidArgument)
 
+>> Text(DateTimeValue(ParseJSON("""2022-05-05""")))
+"5/5/2022 12:00 AM"
+
+>> DateDiff(DateValue(ParseJSON("""2011-01-15T08:00:00.000Z""")), DateValue(ParseJSON("""2011-01-30T08:00:00.000Z""")))
+15
+
+>> DateDiff(DateValue(ParseJSON("""2011-01-15T08:00:00Z""")), DateValue(ParseJSON("""2011-01-30T08:00:00.000""")))
```

Comment:
>DateDiff(DateValue(ParseJSON("""2011-01-15T08:00:00Z""")), DateValue(ParseJSON("""2011-01-30T08:00:00.000"""))) [](http://example.com/codeflow?start=3&length=111)

Same comment as in the test for DateTimeValue - it will fail in certain time zones.

Reply:
Removed tests which used Z, which will fail in some time zones